.yamato/config/_abv.metafile

```diff
@@ -12,6 +12,16 @@ abv: # specify the editor versions for which to create the expression trigger fo
     - il2cpp
   color_spaces: 
     - Linear
+nightly:
+  extra_dependencies: # additional project jobs to run nightly
+    - project: URP_Performance_BoatAttack
```

Comment:
I can't find where these pipelines were _actually_ added in the nightly parent in _abv.yml

Reply:
fixed